extensions/access_log_policy/Makefile

```diff
@@ -1,17 +0,0 @@
-DOCKER_SDK=/sdk
```

Comment:
Why is this removed?

Reply:
This filter uses flat hash map which cannot be directly compiled to wasm. I've confirmed with @gargnupur and @mandarjog, it is not a priority to compile this filter into wasm, so remove it from wasm compilation target for now.